counterpartylib/lib/blocks.py

```diff
@@ -874,11 +874,21 @@ def follow(db):
         block_index = config.BLOCK_FIRST
     else:
         block_index = util.CURRENT_BLOCK_INDEX + 1
+        version_major, version_minor = util.get_user_version()
+        version_changed = False
```

Comment:
yeah you right, but in this case it becomes difficult to add a message in the log when there is a version change.